mscore/musescore.cpp

```diff
@@ -5048,11 +5048,11 @@ int main(int argc, char* av[])
 
       MScore::init();                                      // initialize libmscore
       if (!MScore::testMode) {
-            QSizeF psf = QPrinter().paperSize(QPrinter::Inch);
+            QSizeF psf = QPrinter().paperSize(QPrinter::Point);
             PaperSize ps("system", psf.width(), psf.height());
             PageFormat pf;
             pf.setSize(&ps);
-            pf.setPrintableWidth(psf.width() - 20.0 / INCH);
+            pf.setPrintableWidth(psf.width() - 20.0 / MMPI);
```

Comment:
That is clearly a typo on my part. Thanks for catching it.  I'll post these fixes in a few  minutes.